app/models/github_fetcher/user.rb

```diff
@@ -0,0 +1,27 @@
+module GithubFetcher
+  class User
+    attr_reader :token
+
+    def initialize(token)
+      @token = token
+    end
+
+    def json
```

Comment:
Sorry, I didn't mean for GithubFetcher::User to have required args for initializing.

```ruby
module GithubFetcher
  class User < Resource
    def initialize(user_name, options = {})
      @api_path = "/user/#{user_name}" #example, i dont know if this is actually appropriate
      @options = {}
    end
  end
end
```